src/application-navigation/index.js

```diff
@@ -1,10 +1,19 @@
-import {
-  titleConfigPropType, navigationItemsPropType, extensionItemsPropType, utilityItemsPropType, userConfigPropType,
-} from './private/utils/propTypes';
+// import {
+//   titleConfigPropType,
+//   navigationItemsPropType,
+//   extensionItemsPropType,
+//   utilityItemsPropType,
+//   userConfigPropType,
+// } from './private/utils/propTypes';
```

Comment:
probably want to remove these

Reply:
We probably don't want to remove these to remain passive. Not sure why they're commented....